### PR TITLE
Feat/add responsiveness

### PR DESCRIPTION
**Pull request by BlertaSunCacti**
add responsiveness for smaller sized screens, update design for larger screens

### CODE REVIEW

File: src/app.css [hunk 1]
```diff
@@ -85,9 +85,11 @@ body {
...
 	font-family: 'Material Icons Outlined';
-	font-variation-settings:
+	line-height: 0px;
+	/* font-size: 30px; */
+	/* font-variation-settings:
 		'FILL' 0,
 		'wght' 400,
 		'GRAD' 0,
-		'opsz' 24;
+		'opsz' 24; */
```
Comment (abdulmth) [inline src/app.css:96]: Remove dead code.

File: tailwind.config.js [hunk 2]
```diff
@@ -16,14 +16,26 @@ export default {
...
+		extend: {
+			screens: {
+			  'xs': '320px',      // Phones
+			  'sm': '640px',      // Tablets
+			  'md': '768px',      // Landscape Tablets
+			  'lg': '1080px',     // Laptops
+			  'xl': '1280px',     // Desktops
+			  '2xl': '1440px',    // Larger desktops
+			  '3xl': '1920px',    // Full HD and large desktops
+			}
```
Comment (abdulmth) [inline tailwind.config.js]: why is this needed? aren't these the default?

File: src/components/curator-actions/ChildBountiesSection.svelte [hunk 3]
```diff
@@ -1,56 +1,126 @@
...
+
+										{#if index < filters.length - 1}
+											<hr />
+										{/if}
+									{/each}
+								</div>
+							</div>
+						{/if}
+					</div>
+				</div>
```
Comment (abdulmth) [inline src/components/curator-actions/ChildBountiesSection.svelte:102]: we can't have the drop down logic here, it should be in its own component.

**Review by abdulmth — CHANGES_REQUESTED** [review]

File: src/app.css (hunk 1 above)
Comment (BlertaSunCacti) [reply src/app.css:96]: Might need them for later when I fix the icons, therefore did not delete them.

File: tailwind.config.js (hunk 2 above)
Comment (BlertaSunCacti) [reply tailwind.config.js]: Not entirely, I could remove the `sm`, `md` and `xl` as they are the same as default, but i overrode the other ones.

File: src/components/curator-actions/ChildBountiesSection.svelte (hunk 3 above)
Comment (abdulmth) [reply src/components/curator-actions/ChildBountiesSection.svelte:102]: let's use the built in dropdown menu for now, we can create a better component later.

File: tailwind.config.js (hunk 2 above)
Comment (abdulmth) [reply tailwind.config.js]: Maybe add a comment then why this is needed.

File: src/app.css (hunk 1 above)
Comment (abdulmth) [reply src/app.css:96]: Add a TODO comment then so it doesn't get lost in the future.

**Review by abdulmth — APPROVED** [review]
